docs/components/modeler/web-modeler/token-simulation.md

```diff
@@ -1,13 +1,15 @@
 ---
 id: token-simulation
 title: Token simulation
-description: You can use the token simulation feature to observe how the process will behave when it is executed.
+description: Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
 ---
 
-You can use the token simulation feature to observe how the process will behave when it is executed.
+Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
```

Comment:
A note that behavior in token simulation may differ from engine execution would be valuable, with some examples (e.g. tokens look different, no implementation details are considered so link events are uncorrelated)

To see how the process would execute on a Zeebe cluster, use Play instead